src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -1767,6 +1768,7 @@ namespace Akka.Actor.Internal
         [System.ObsoleteAttribute("Use WhenTerminated instead. This property will be removed in future versions")]
         public override System.Threading.Tasks.Task TerminationTask { get; }
         public override System.Threading.Tasks.Task WhenTerminated { get; }
+        public override void Abort() { }
```

Comment:
Concrete implementation of `ExtendedActorSystem.Abort` lives on the `ActorSystemImpl` class, which is why this shows up twice in the approval file.